### ISSUE TITLE
Error: Building MobileMessaging.framework with Carthage

### ISSUE THREAD

**Issue by phvillegas** [infobip/mobile-messaging-react-native-plugin#20]
Hello team,

I've got some errors when building `MobileMessaging.framework`.

I upgraded  Xcode and when I run `npm run mm_build` in `node_modules/infobip-mobile-messaging-react-native-plugin`.
I could see the following error in the terminal:

```
Building MobileMessaging.framework with Carthage started...
*** Checking out mobile-messaging-sdk-ios at "9.0.1"
*** No cache found for mobile-messaging-sdk-ios, building with all downstream dependencies
*** xcodebuild output can be found in /var/folders/pb/34w99wj969g1xrh0dg2mhs0c0000gn/T/carthage-xcodebuild.SdnK2h.log
*** Building scheme "MobileMessaging" in MobileMessaging.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/phvillegas/app/node_modules/infobip-mobile-messaging-react-native-plugin/ios/Carthage/Checkouts/mobile-messaging-sdk-ios/MobileMessaging.xcodeproj -scheme MobileMessaging -configuration Release -derivedDataPath /Users/phvillegas/Library/Caches/org.carthage.CarthageKit/DerivedData/12.5_12E262/mobile-messaging-sdk-ios/9.0.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive VALIDATE_WORKSPACE=NO -archivePath /var/folders/pb/34w99wj969g1xrh0dg2mhs0c0000gn/T/mobile-messaging-sdk-ios SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/phvillegas/app/node_modules/infobip-mobile-messaging-react-native-plugin/ios/Carthage/Checkouts/mobile-messaging-sdk-ios)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/pb/34w99wj969g1xrh0dg2mhs0c0000gn/T/carthage-xcodebuild.SdnK2h.log
Building MobileMessaging.framework with Carthage finished.
/Users/phvillegas/app/node_modules/infobip-mobile-messaging-react-native-plugin
```

### package.json:

```json
"react-native": "0.63.4",
"infobip-mobile-messaging-react-native-plugin": "^2.6.0"
```

Hope someone can help me.
Thanks!

**Comment by riskpp** [comment]
Hi @phvillegas, thank you for reporting it, I think it's Xcode 12.5 compatibility issue, as I see from logs you are using this version. We will update plugin soon.

BR, Olga

**Comment by riskpp** [comment]
@phvillegas, we've released [3.1.0 version](https://github.com/infobip/mobile-messaging-react-native-plugin/releases/tag/3.1.0) with Xcode 12.5 support.

BR, Olga

**Comment by phvillegas** [comment]
Hi @riskpp 

Thanks for your support.

The latest version works for me.

Regards.
